packages/browser/src/node/commands/keyboard.ts

```diff
@@ -129,8 +129,7 @@ export async function keyboardImplementation(
 
     for (const { releasePrevious, releaseSelf, repeat, keyDef } of actions) {
       let key = keyDef.key!
-      const code = 'location' in keyDef ? keyDef.key! : keyDef.code!
-      const special = Key[code as 'Shift']
```

Comment:
I changed this as it wasn't working when `key === 'Ctrl'`, which is webdriverio's special key to switch modifier keys based on platform. I'm not sure the original code's intent, but it didn't break existing tests.

Reply:
The difference here is `AltrRight` for example - see https://github.com/testing-library/user-event/blob/main/src/keyboard/keyMap.ts. Without this, it will always trigger `Alt`, not `AltRight`

Reply:
This is broken anyways regardless of what I do here. (see `test/browser/fixtures/user-event/keyboard.test.ts`. on main `[Shift]` works differently, but `[Shift]` shouldn't actually work).

I don't think solving this is a scope of PR adding copy/cut/paste sugars, so we can follow this up separately https://github.com/vitest-dev/vitest/issues/7118.